src/lib/operators/insert.cpp

```diff
@@ -228,13 +228,6 @@ std::shared_ptr<const Table> Insert::_on_execute(std::shared_ptr<TransactionCont
     start_index = 0u;
   }
 
-  if (transaction_context_is_set()) {
-    if (!constraints_satisfied(_target_table_name, transaction_context()->snapshot_commit_id(),
-                                   transaction_context()->transaction_id())) {
-      _mark_as_failed();
-    }
-  }
-
```

Comment:
What happened to this?

Reply:
What do you mean? The check if the transaction context is gone, as it is already asserted in the constructor of the operator.

Reply:
Ah, ok. Now I remember reading that suggestion.